classes/query/QueryParticipant.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file classes/query/QueryParticipant.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class QueryParticipant
+ *
+ * @brief Class for QueryParticipant.
+ */
+
+namespace PKP\query;
+
+use APP\facades\Repo;
+use Eloquence\Behaviours\HasCamelCasing;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class QueryParticipant extends Model
+{
+    use HasCamelCasing;
+
+    protected $table = 'query_participants';
+    protected $primaryKey = 'query_participant_id';
+    public $timestamps = false;
+
+    protected $fillable = [
+        'queryId', 'userId'
+    ];
+
+    protected function casts(): array
+    {
+        return [
+            'queryId' => 'int',
+            'userId' => 'int'
+        ];
+    }
+    public function toQuery(): BelongsTo
```

Comment:
suggestion: you can rename  `toQuery()` to `query()` to align with Laravel's naming conventions for relationships.

Reply:
I didn't follow the convention in this case because `query()` [is already a method in the `Model` class](https://laravel.com/api/11.x/Illuminate/Database/Eloquent/Model.html#method_query).